lib/apisauce.ts

```diff
@@ -285,6 +292,18 @@ export const create = config => {
       forEach(transform => transform(transformedResponse), responseTransforms)
     }
 
+		// add the async response transforms
+		if (asyncResponseTransforms.length > 0) {
```

Comment:
This section is still not aligned correctly. There is too much whitespace along the left margins. Please ensure that it is spaced like the rest of the code base.